vignettes/pkgnet-intro.Rmd

```diff
@@ -0,0 +1,354 @@
+
+```{r whatIsDepGraph, echo=FALSE, message=FALSE, results='markup'}
+nodes <- data.frame(
+  id = 1:4
+  , label = LETTERS[1:4]
+  , x = c(1,3,2,4)
+  , y = c(1,1,2,3)
+  , level = c(1,1,2,3)
+  )
+
+edges <- data.frame(
+  from = c(1,2,3,2)
+  , to = c(3,3,4,4)
+  )
+
+g <- visNetwork::visNetwork(nodes = nodes
+                       , edges = edges
+                       , width = "100%"
+                       )
+g <- visNetwork::visNodes(graph = g
+                          , shape = "circle"
+                          , font = list(size = 25
+                                        , bold = TRUE
+                                        , align = 'center'
+                                        )
+                       )
+g <- visNetwork::visEdges(graph = g
+                          , arrows = "to"
+                       )
+g <- visNetwork::visHierarchicalLayout(graph = g
+                      , direction = "DU"
+                      )
+g <- visNetwork::visInteraction(graph = g
+                                , dragNodes = TRUE
+                                , dragView = TRUE
+                                , zoomView = FALSE)
+g
+```
+
+
+***
+# Examining The Results
+
+An HTML report has been created with the pertinent information and a list object is available with the same information and more.  
+
+## The Report
+
+An HTML report has been created, and its location is specified in the messages in the terminal.  
+
+This report has three sections: 
+
+1. **Package Summary** -- general information about the package and package level statistics   
+2. **Dependency Network** -- information regarding the packages upon which the current package under analysis relies upon    
+3. **Function Network** -- information regarding the functions within the current package under analysis and their interdependencies     
+
+Each section has helpful tables and visuals.  
+
+As a sample, here's how the **Function Network Visualization** looks for `pkgnet`: 
+
+<!--  SIDE BY SIDE START  -->
+<!--  Take care to ensure equal lengths  -->
+<div class="column-left">
+#### Default
+```{r demoVis1, echo=FALSE}
+report1$PackageFunctionReporter$graph_viz
+```
+Functional dependencies are visible.  For example, we can see that both `batting_avg` and `slugging_avg` functions depend upon the `at_bats` function.  
+
+We also see that nothing depends on the `on_base_pct` function.  This might be valuable information to an R package developer. 
+</div>
+
+<div class="column-right">
+#### With Coverage Information 
+```{r demoVis2, echo=FALSE}
+report2$PackageFunctionReporter$graph_viz
+```
+Same as the default visualization except we can see coverage information as well (Red = 0%, Green = 100%). 
+
+It appears the function that most other functions depend upon, `at_bats`, is well covered.  However, no other functions are covered by unit tests.
+</div>
+
+<!--  SIDE BY SIDE END -->
+
+<br><br>
+
+#### Check out the full HTML report for more results  
```

Comment:
I'm seeing some weird layout thing going on in the rendered HTML. 
<img width="1016" alt="screen shot 2018-03-30 at 7 32 12 pm" src="https://user-images.githubusercontent.com/2721979/38157903-265e5d36-3451-11e8-8c76-8908e56176b0.png">